src/frontend/screens/Observation/PresetHeader.tsx

```diff
@@ -8,7 +8,7 @@ import {Preset} from '@mapeo/schema';
 export const PresetHeader = ({preset}: {preset: Preset}) => {
   return (
     <View style={styles.categoryIconContainer}>
-      <PresetCircleIcon size="medium" name={preset.name} />
+      <PresetCircleIcon size="medium" name={preset.name} testID={preset.name} />
```

Comment:
ah this might be needed for asserting that the icon is present. if that's the case, maybe update the id to indicate that it's an icon e.g. ``icon-${preset.name}``

Reply:
I got rid of it. Was a mistake I think